homeassistant/components/plum_lightpad/strings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Plum Lightpad",
```

Comment:
Don't set title to be translatable if it is the same as title in manifest. Just remove it.
```suggestion
```